src/utilities/geometry/Test/ThreeJS_GTest.cpp

```diff
@@ -51,6 +51,19 @@ TEST_F(GeometryFixture, ThreeJS) {
   boost::optional<ThreeScene> scene = ThreeScene::load(contents);
   ASSERT_TRUE(scene);
 
+  // checking user data
+  std::vector<ThreeSceneChild> sceneChildren = scene->object().children();
+  for (const ThreeSceneChild& sceneChild : sceneChildren) {
+    EXPECT_DOUBLE_EQ(1, sceneChild.userData().airLoopHVACNames().size());
```

Comment:
Can we change to `EXPECT_DOUBLE_EQ(1, double(sceneChild.userData().airLoopHVACNames().size()));`

windows is throwing the following due to treating warning as error when compiling. 

```
D:\jenkins\openstudio\develop\src\utilities\geometry\Test\ThreeJS_GTest.cpp(57,5): error C2220: the following warning is treated as an error [D:\jenkins\openstudio\develop\build\src\utilities\openstudio_utilities_tests.vcxproj]
D:\jenkins\openstudio\develop\src\utilities\geometry\Test\ThreeJS_GTest.cpp(57,5): warning C4244: 'argument': conversion from 'unsigned __int64' to 'RawType', possible loss of data [D:\jenkins\openstudio\develop\build\src\utilities\openstudio_utilities_tests.vcxproj]
```

Reply:
Nah, you want `EXPECT_EQ(1, xxx.size());` (or `EXPECT_EQ(1u, xxx.size());` )

Reply:
Even better!